docs/design/pattern_matching.md

```diff
@@ -718,3 +742,15 @@ and/or templates.
 
```

Comment:
(Can't comment on the right line, sorry.)

Can we remove the "Generic/template pattern matching" section above?